etcdserver/api/v2v3/store_test.go

```diff
@@ -66,6 +66,10 @@ func TestCreateKV(t *testing.T) {
 		}
 
 		evg, err := v2.Get(tc.key, false, false)
+		if err != nil {
+			log.Fatal(err)
```

Comment:
Can we `t.Fatal`?

Reply:
I'd be happy to. I was emulating what the function is doing on line 46. Should I change that to `t.Fatal()` as well?

Reply:
@alrs yep, please also change line 46. thanks.

Reply:
OK, made that change and two more to get `log` out of `store_test.go` entirely.